### PR TITLE
JAVA-39455:Changes made for upgrade spring-thymeleaf from spring 5 to…

### PR DESCRIPTION
**Pull request by Bipinkumar27**
… spring 6

### CODE REVIEW

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package com.baeldung.thymeleaf.config;
 
-import javax.servlet.ServletRegistration.Dynamic;
+
```
Comment (alvin577austria) [inline spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java:3]: Can you remove the empty lines here please.

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java [hunk 2]
```diff
@@ -29,8 +31,7 @@ protected String[] getServletMappings() {
     }
 
     @Override
-    protected void customizeRegistration(final Dynamic registration) {
-        super.customizeRegistration(registration);
+    protected void customizeRegistration(ServletRegistration.Dynamic registration) {
```
Comment (alvin577austria) [inline spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java:34]: It seems we are overriding this method but just calling the parent inside. Can we remove this so we just simply inherit the one from parent?

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebMVCSecurity.java [hunk 3]
```diff
@@ -27,12 +27,12 @@ public InMemoryUserDetailsManager userDetailsService() {
 
     @Bean
     public WebSecurityCustomizer webSecurityCustomizer() {
-        return (web) -> web.ignoring().antMatchers("/resources/**");
+        return (web) -> web.ignoring().requestMatchers("/resources/**");
```
Comment (alvin577austria) [inline spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebMVCSecurity.java:30]: I guess the parentheses around web can be removed.

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java (hunk 1 above)
Comment (Bipinkumar27) [reply spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java]: Done

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java (hunk 2 above)
Comment (Bipinkumar27) [reply spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java]: Done

File: spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebMVCSecurity.java (hunk 3 above)
Comment (Bipinkumar27) [reply spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebMVCSecurity.java]: correct, I have done the changes

**Review by alvin577austria — APPROVED** [review]

**Comment by alvin577austria** [comment]
retest this please.

**Review by alvin577austria — APPROVED** [review]

**Pull request merged**